HW02/Vadim_Volovenko/vadim_volovenko_hw_02_t_01.rb

```diff
@@ -0,0 +1,8 @@
+logs = File.readlines('data_for_t_01.txt', chomp: true)
+
+def task_1(logs)
+  error_message = logs.find { |log| log.downcase.include?('error') }
+  error_message.empty? ? '' : error_message
```

Comment:
You can try to use the shortest version
```suggestion
  logs.find { |log| log.downcase.include?('error') }.to_s
```